jsk_topic_tools/include/jsk_topic_tools/connection_based_nodelet.h

```diff
@@ -91,6 +91,12 @@ namespace jsk_topic_tools
     virtual void connectionCallback(const ros::SingleSubscriberPublisher& pub);
 
     /** @brief
+     * callback function which is called when walltimer
+     * duration run out.
+     */
+    virtual void wallTimerCallback(const ros::WallTimerEvent& event);
```

Comment:
name of the method should be `warnConnectionOnceCallback` or somtehing like that